src/main/java/org/alfresco/opencmis/CMISConnector.java

```diff
@@ -4109,5 +4089,30 @@ private CMISRenditionMapping getRenditionMapping()
             singletonCache.put(KEY_CMIS_RENDITION_MAPPING_NODEREF, renditionMapping);
         }
         return renditionMapping;
+    }
+
+    private boolean isUpdatable(Updatability updatability, boolean isOnWorkingCopy)
+    {
+        if ((updatability == Updatability.READONLY)
+                || (updatability == Updatability.WHENCHECKEDOUT && !isOnWorkingCopy))
+        {
+            return false;
+        }
+        else
+        {
+            return true;
+        }
+    }
+
+    private boolean isUpdatable(Updatability updatability)
```

Comment:
There is no need to **overload** the _isUpdatable_ method.
Just use Boolean instead of boolean and allow true/false/null.
Also, methods that are inherited will inherit the javadoc as well.
Private methods do not have that luxury: you should add javadoc for the private method to say what it does and how the arguments are used.